hadoop-hdds/framework/src/test/java/org/apache/hadoop/hdds/utils/db/TestRDBTableStore.java

```diff
@@ -96,11 +96,7 @@ public static void initConstants() {
 
   private static boolean consume(Table.KeyValue keyValue)  {
     count++;
-    try {
-      assertNotNull(keyValue.getKey());
-    } catch (IOException ex) {
-      fail("Unexpected Exception " + ex);
-    }
+    assertDoesNotThrow(() -> assertNotNull(keyValue.getKey()));
```

Comment:
Since both `assertNotNull` and `getKey()` can throw, I think it's better to reverse order of assertions:

```suggestion
    assertNotNull(assertDoesNotThrow(keyValue::getKey));
```